### PR TITLE
Maquetado navbar

### PR DESCRIPTION
**Pull request by andyOlarte514**
# Summary
- add the code of the navbar component to the `index.html`
- create archive `navbar.scss` and import to `application.scss`

# Trello Card
- https://trello.com/c/yo1dWvhM

# Screenshots
navbar

### CODE REVIEW

File: training-maquetado/index.html [hunk 1]
```diff
@@ -8,6 +8,14 @@
   <link rel="stylesheet" type="text/css" href="application.css">
 </head>
 <body>
+  <header class="navbar d-flex align-center justify-between">
+    <div class="navbar-logo">
```
Comment (CarlosHWolox) [inline training-maquetado/index.html]: I think you can remove this div and add this class directly to the child element

File: training-maquetado/index.html [hunk 2]
```diff
@@ -8,6 +8,14 @@
...
+      <img src="./assets/wolox_logo.svg" alt="Wolox Logo">
+    </div>
+    <div class="navbar-subtitle">
```
Comment (CarlosHWolox) [inline training-maquetado/index.html]: same here

File: training-maquetado/scss/navbar.scss [hunk 3]
```diff
@@ -0,0 +1,26 @@
+.navbar {
+  position: fixed;
+  top: 0;
+  width: 100%;
+  max-width: 100%;
+  height: 90px;
```
Comment (CarlosHWolox) [inline training-maquetado/scss/navbar.scss]: There was an already defined variable for the height of the navbar at the beginning of the training

File: training-maquetado/scss/navbar.scss [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+  padding: 0 30px;
+  background-color: $white;
+  z-index: 2;
+  box-shadow: 0 5px 10px $grey;
+
+  .navbar-logo {
+    height: 30px;
```
Comment (CarlosHWolox) [inline training-maquetado/scss/navbar.scss]: I think for this one there was one aswell

File: training-maquetado/scss/navbar.scss [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+
+    img{
+      height: 100%;
+    }
+  }
+
+  .navbar-subtitle {
+
```
Comment (CarlosHWolox) [inline training-maquetado/scss/navbar.scss]: delete line

File: training-maquetado/scss/navbar.scss [hunk 6]
```diff
@@ -0,0 +1,26 @@
...
+    p{
```
Comment (CarlosHWolox) [inline training-maquetado/scss/navbar.scss]: space

File: training-maquetado/scss/navbar.scss [hunk 7]
```diff
@@ -0,0 +1,26 @@
...
+  height: 90px;
+  padding: 0 30px;
+  background-color: $white;
+  z-index: 2;
+  box-shadow: 0 5px 10px $grey;
+
+  .navbar-logo {
+    height: 30px;
+
+    img{
```
Comment (CarlosHWolox) [inline training-maquetado/scss/navbar.scss]: space

**Review by CarlosHWolox — DISMISSED** [review]

File: training-maquetado/scss/techs.scss [hunk 8]
```diff
@@ -3,7 +3,7 @@
   padding: 0 70px;
 
   p {
```
Comment (braiandickson) [inline training-maquetado/scss/techs.scss:5]: Try to style the classes insted html tags.

**Review by braiandickson — CHANGES_REQUESTED** [review]

**Review by braiandickson — APPROVED** [review]

**Pull request merged**